From: Dmitry Shachnev <mitya57@debian.org>
Date: Tue, 2 Jun 2020 12:22:52 +0300
Subject: Prevent Sphinx from downloading intersphinx indexes

Sphinx wants to download interspinx objects such as
https://docs.python.org/3/objects.inv
during build.
---
 doc/source/conf.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index a097886..8b13f1c 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -59,8 +59,8 @@ source_suffix = '.rst'
 qt_documentation = "Qt6"
 
 intersphinx_mapping = {
-    'python': ('https://docs.python.org/3', None),
-    'numpy': ('https://numpy.org/doc/stable/', None)
+    'python': ('/usr/share/doc/python3-doc/html', None),
+    'numpy': ('/usr/share/doc/python-numpy/html', None)
 }
 
 nitpick_ignore_regex = [
